src/main/java/com/hillel/items_exchange/dao/ImageRepository.java

```diff
@@ -0,0 +1,18 @@
+package com.hillel.items_exchange.dao;
+
+import com.hillel.items_exchange.model.Image;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ImageRepository extends JpaRepository<Image, Long> {
+
+    @Query(value = "SELECT resource_url FROM image WHERE  product_id = ?1", nativeQuery = true)
+    List<String> findImageUrlsByProductId(Long id);
+
+    @Query(value = "SELECT id, product_id, resource_url, default_photo FROM image WHERE  product_id = ?1", nativeQuery = true)
```

Comment:
Queries into this repository are quite easy. You could get rid of them end use Spring Data Expression language to create requests with key words: 
https://docs.spring.io/spring-data/keyvalue/docs/current/reference/html/#repository-query-keywords

Reply:
I wonder if we do need findImageUrlsByProductId(Long id) or not?

Reply:
Lest keep is for now.

Reply:
I rewrote those requests. As for me now they aren't easy enough to be removed.